pkg/minishift/setup/platform/platformsetup_windows.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package platform
+
+import (
+	"fmt"
+	"github.com/minishift/minishift/pkg/minishift/shell/powershell"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	GetGroupCmd = `(New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")).Translate([System.Security.Principal.NTAccount]).Value`
+)
+
+// Default External Virtual Switch Name
+const ExternalVirtualSwitchName = "minishift-ext-switch"
```

Comment:
suggestion s/minishift-ext-switch/minishift-external/ no need to add switch to the name as it is understood.